src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -355,6 +355,34 @@ SendCommandListToWorkerOutsideTransaction(const char *nodeName, int32 nodePort,
 }
 
 
+/*
+ * SendCommandListToWorkerInSeparateTransaction sends the command list over the
+ * connection in a separate connection. This opens a new transaction on the
+ * connection, thus it's important that no transaction is currently open on the
+ * given connection. This function is mainly useful to avoid opening an closing
+ * connections excessively by allowing reusing a single connection to send
+ * multiple separately committing transactions. The function raises an error if
+ * any of the queries fail.
+ */
+void
+SendCommandListToWorkerInSeparateTransaction(MultiConnection *workerConnection,
```

Comment:
I like that we pass `MultiConnection *` otherwise it'd be very hard to remember we should close this connection